src/go/configmanager/config_manager.go

```diff
@@ -51,10 +51,10 @@ var (
 // Config Manager handles service configuration fetching and updating.
 // TODO(jilinxia): handles multi service name.
 type ConfigManager struct {
-	serviceName               string
-	serviceInfo               *configinfo.ServiceInfo
-	envoyConfigOptions        options.ConfigGeneratorOptions
-	configIdServiceConfigFile string
+	serviceName                   string
+	serviceInfo                   *configinfo.ServiceInfo
+	envoyConfigOptions            options.ConfigGeneratorOptions
+	configIdFromServiceConfigFile string
```

Comment:
Hmm,  I am wondering if we can define an interface to share between ServiceConfigFile and ServiceConfigFetcher.  ConfigMgr only use the interface.